### ISSUE TITLE
Create a release

### ISSUE THREAD

**Issue by maxlerebourg** [poundifdef/SmoothMQ#9]
Hey, I'm coming from HN.

To help the community to test your tool, could you create a release on github or a docker image.

Good luck with this project.

**Comment by poundifdef** [comment]
I agree. The tricky part is automating this with cgo for different platforms.

**Comment by diogox** [comment]
I haven't used it, but my understanding [GoReleaser](https://goreleaser.com/) is the tool typically used for this. There's also a [Github Action](https://github.com/goreleaser/goreleaser-action) for it 🙌

**Comment by huksley** [comment]
Would be good for making a release and also tagging a repo to the latest version? Which includes changes #16  

Also good to have a Homebrew recipe to use it.

**Comment by huksley** [comment]
For example https://gist.github.com/huksley/d88da22046c29d34d9193a602e3e6661
